### ISSUE TITLE
[Feature]: More options when create "New note from card"

### ISSUE THREAD

**Issue by GustavoSZ124** [mgmeyers/obsidian-kanban#873]
### Goal or desired outcome of this feature

Some settings and options when creating notes from cards:

1. Open new note: Off/On
2. When open new note On: in a new tab/to the right
3. When open new note On, Focus on new note: Off/On
4. Optional new file name

### Describe the feature

Tabs are mostly used since their appearance (Obsidian 1.0) and should be the default option when creating a note from a card, plus the panels take up space from the Kanban board in the window. But it is better to have the options for people who continue using the boards or who for whatever reason do not want the new note to open, such as putting the focus on the new note or staying on the Kanban board.

It would also be very useful to have the option to name the new note and use the card text as an alias *[[new name|text of cart]]* since it usually contains special characters or does not follow one's naming conventions.

### Can you think of any alternatives or work-arounds?

Everything can be done manually but it is tedious and cumbersome.

**Comment by HugleDong** [comment]
Agreed. We're eager for this feature! Every time a new note is created from kanban, the original layout is broken.
